packages/grid/data-grid/src/DataGrid.tsx

```diff
@@ -77,6 +80,18 @@ DataGrid2.propTypes = {
       );
     }
   }),
+  disableColumnReorder: chainPropTypes(PropTypes.bool, (props) => {
+    if (props.disableColumnReorder === false) {
+      throw new Error(
+        [
+          `Material-UI: \`<DataGrid disableColumnReorder={false} />\` is not a valid prop.`,
+          'Column reordering is not available in the MIT version',
```

Comment:
```suggestion
          'Column reordering is not available in the MIT version.',
```

Reply:
Done. Full stops were also missing on some of the other sentences -> I added there as well